src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/Hand.java

```diff
@@ -61,4 +63,40 @@ public Participation getParticipation() {
   public void setParticipation(Participation participation) {
     this.participation = participation;
   }
+
+  public boolean isTurnActive() {
```

Comment:
If someone has a better name, i am open for suggestions

Reply:
`turnIsActive` oder aber ich finde auch `isTurnActive` gut